crates/zizmor/src/audit/dependabot_execution.rs

```diff
@@ -13,6 +14,24 @@ audit_meta!(
 
 pub(crate) struct DependabotExecution;
 
+impl DependabotExecution {
+    /// Creates a fix that changes insecure-external-code-execution from allow to deny
+    fn create_set_deny_fix<'doc>(update: crate::models::dependabot::Update<'doc>) -> Fix<'doc> {
+        Fix {
+            title: "set insecure-external-code-execution to deny".to_string(),
+            key: update.location().key,
+            disposition: FixDisposition::Safe,
```

Comment:
Nit: We should probably make this one as unsafe for now, since I imagine people often *do* have this set for load-bearing reasons 🙂